geniza/corpus/templates/corpus/document_detail.html

```diff
@@ -70,11 +70,27 @@ <h2 class="sr-only">
                 {% endif %}
             </dl>
             {# secondary metadata #}
-            {% if document.languages.exists or document.secondary_languages.exists %}
+            {% if document.languages.exists or document.secondary_languages.exists or document.doc_date_original or document.doc_date_standard %}
                 <dl class="metadata secondary">
-                    {# NOTE: placeholder to populate translation string #}
-                    {# Translators: label for date of this document, if known #}
-                    <!-- {% translate "Document Date" %} -->
+                    {% if document.doc_date_original or document.doc_date_standard %}
+                        <dt>
+                            {# Translators: label for date of this document, if known #}
+                            {% translate "Document Date" %}
+                        </dt>
+                        <dd>
+                            {% if document.doc_date_original %}
+                                <time{% if document.doc_date_standard %} datetime="{{ document.doc_date_standard }}"{% endif %}>
+                                    {{ document.historical_date_display }}{% if document.doc_date_standard %}
+                                        ({{ document.doc_date_standard }})
+                                    {% endif %}
+                                </time>
+                            {% else %}
+                                <time datetime="{{ document.doc_date_standard }}">
+                                    {{ document.doc_date_standard }}
+                                </time>
+                            {% endif %}
```

Comment:
This looks more complicated than I want in a template. The two different cases are because the standardized date needs to be parenthesized if we have a historical date? 

Let's shift the logic to a document method — how about just calling it `document_date`? (I think we'll probably need this for indexing in Solr anyway.)

Should we add `CE` for the standardized date? That seems to be the convention in the dates I see in descriptions, and I think would make the date information more legible.